crates/krator/src/runtime.rs

```diff
@@ -90,16 +113,16 @@ impl<O: Operator> OperatorRuntime<O> {
       skip(self, event),
       fields(event = ?PrettyEvent::from(&event))
     )]
-    async fn dispatch(&mut self, event: Event<O::Manifest>) -> anyhow::Result<()> {
-        match &event {
-            Event::Applied(object) => {
-                let key: ObjectKey = object.into();
+    async fn dispatch(&mut self, event: ObjectEvent<O::Manifest>) -> anyhow::Result<()> {
+        match event {
+            ObjectEvent::Applied(object) => {
+                let key: ObjectKey = (&object).into();
                 // We are explicitly not using the entry api here to insert to avoid the need for a
                 // mutex
                 match self.handlers.get_mut(&key) {
```

Comment:
a `get` should be enough here, shouldn't it?

Reply:
I think you are right, Tokio 0.3 made it so you don't need a mutable reference when calling `Sender::send`. Surprised Clippy doesnt complain.